pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -217,13 +217,13 @@ func unpublishDeletedNetworkInstanceConfig(ctx *getconfigContext,
 		networkInstanceEntry := lookupNetworkInstanceById(key, networkInstances)
 		if networkInstanceEntry != nil {
 			// Entry not deleted.
-			log.Infof("NetworkInstance %s (Name: %s) still exists\n",
+			log.Infof("NetworkInstance %s (Name: %s) still exists",
 				key, networkInstanceEntry.Displayname)
 			continue
 		}
 
 		config := entry.(types.NetworkInstanceConfig)
-		log.Infof("unpublishing NetworkInstance %s (Name: %s) \n",
+		log.Infof("unpublishing NetworkInstance %s (Name: %s) ",
```

Comment:
Would be good to remove the trailing space as well

Reply:
Resolved